test/Microsoft.ML.Functional.Tests/Common.cs

```diff
@@ -20,5 +20,34 @@ public static void CheckMetrics(RegressionMetrics metrics)
             Assert.True(metrics.L2 >= 0);
             Assert.True(metrics.RSquared <= 1);
         }
+
+        public static void AssertEqual(float[] array1, float[] array2)
```

Comment:
>AssertEqual [](start = 27, length = 11)

Perhaps Enumerable.SequenceEqual(target1, target2); instead? from Linq

Reply:
We need to use the assert logic here so that we can capture equality to some level of precision. Default equality will fail after serialization due to floating point drift sometimes.

---
In reply to: [256208383](https://github.com/dotnet/machinelearning/pull/2518#discussion_r256208383) [](ancestors = 256208383)

Reply:
Plus I want to Assert and not check.

---
In reply to: [256209571](https://github.com/dotnet/machinelearning/pull/2518#discussion_r256209571) [](ancestors = 256209571,256208383)